README.md

```diff
@@ -39,85 +39,78 @@ This repository contains a cross-platform sample project demonstrating all the p
 
 ![The Vortex Sandbox app demonstrating several built-in particle presets.](sandbox-preview.gif)
 
+There are also a number of Xcode previews created for a selected number of the presets contained in the Presets folder.
 
 ## Basic use
 
-Rendering a Vortex particle system takes two steps:
-
-1. Creating an instance of `VortexSystem`, configured for how you want your particles to behave. This must be given a list of tag names of the particles you want to render.
-2. Adding a `VortexView` to your SwiftUI view hierarchy, passing in the particle system to render, and also all the views that are used for particles, tagged using the same names from step 1.
-
-There are lots of built-in particle system designs, such as rain:
-
+Rendering a Vortex particle system using a preset configuration takes a single line! 
 ```swift
-VortexView(.rain) {
-    Circle()
-        .fill(.white)
-        .frame(width: 32)
-        .tag("circle")
-}
+VortexView(.rain) 
 ```
 
-Fireworks:
-
-```swift
-VortexView(.fireworks) {
-    Circle()
-        .fill(.white)
-        .blendMode(.plusLighter)
-        .frame(width: 32)
-        .tag("circle")
-}
-```
+There are lots of built-in particle system designs, such as Fireworks (.fireworks) and Fire (.fire)
 
-And fire:
 
+You can also easily create custom effects, in two steps:
+1. Creating an instance of `VortexSystem.Settings`, configured for how you want your particles to behave. The easiest way to create Settings is to modify an existing one. (example below)
+2. Call `VortexView` with the settings you just created.
+e.g. 
 ```swift
-VortexView(.fire) {
-    Circle()
-        .fill(.white)
-        .blendMode(.plusLighter)
-        .blur(radius: 3)
-        .frame(width: 32)
-        .tag("circle")
+struct ContentView: View {
+    var body: some View {
+        let fireSettings = VortexSystem.Settings(from: .fire ) { settings in
```

Comment:
It's generally not a good idea to mix up logic and layout in the `body` property, so I think for the README it might be a better idea to pull this settings object out into its own property.